### PR TITLE
Docs/setup

### PR DESCRIPTION
**Pull request by Caposto**
Added instructions to the README.md & env.example.

I had an issue running the Docker container. It would return a 500 internal server error and it seemed like the Auth and tRPC connections weren't working. However, running the application with pnpm run dev worked.

I was also thinking of adding a dev mode to disable authentication w/ another environment variable. Was looking into it, but wondering what everyone else thinks first.

### CODE REVIEW

**Review by ericzty — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,9 +1,52 @@
...
+   [Installing pnpm](https://pnpm.io/installation).
+
+   Run `pnpm install` in your terminal in the root directory of the project
+
+4. **Start & Seed Prisma**
+
+   Run the following two commands in your terminal:
+
+   ```
+   pnpm run postinstall
```
Comment (ulrokx) [inline README.md:23]: This should run automatically after install, might not need this.

File: src/pages/index.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ function AuthShowcase() {
     <div className="mb-20 flex flex-col items-center justify-center gap-4">
       <p className="text-center text-2xl text-white">
         {sessionData && <span>Logged in as {sessionData.user?.name}</span>}
-        {/* {secretMessage && <span> - {secretMessage}</span>} */}
+        {/*<meta name="google-signin-client_id" content="YOUR_CLIENT_ID.apps.googleusercontent.com" />*/}
```
Comment (ulrokx) [inline src/pages/index.tsx:49]: If we take this out does this still work? This seems like something that NextAuth probably already does for us.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,52 @@
...
 
-2. specify client id by including this line:
-<meta name="google-signin-client_id" content="YOUR_CLIENT_ID.apps.googleusercontent.com">
+1. **Create and setup your .env file**: Refer to .env.example
 
-3. added a new google-signin.tsx (not sure if this necessary, just following along with documentation)
+2. **Add Google OAuth 2.0 client to the Application**: in index.ts (src/pages/index.tsx) add the following to _AuthShowcase_
+
+   ```
+   <meta name="google-signin-client_id" content="YOUR_CLIENT_ID.apps.googleusercontent.com" />
```
Comment (ulrokx) [inline README.md:10]: I think supplying the env var should be enough, we should not need to do this. For my own project with NextAuth I did not need to do this and it was able to auth through google.

**Review by ulrokx — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Caposto) [reply README.md]: You are right, I removed and just had in .env and it worked.

**Pull request merged**